assets/css/navigation.css

```diff
@@ -0,0 +1,258 @@
+/* Header section */
+@font-face {
+    font-family: KaushanScript;
+    src: url("../fonts/KaushanScript-Regular.otf") format("opentype");
```

Comment:
Where is this used?

Reply:
Used this font for "Doc2Pen" in the left.